1172 publish the tags we use for tracking poor public body performance on the help pages

HelenWDTK opened the pull request:
## Relevant issue(s)
Fixes #1172
## What does this do?
Adds a help page with the performance tag table on it.
## Why was this needed?
We wanted to be able to publicly link to the tags we use, so we can publicly monitor how widespread an issue is.
## Implementation notes
N/A
## Screenshots
<img width="1202" alt="Screenshot 2023-06-08 at 10 15 59" src="https://github.com/mysociety/whatdotheyknow-theme/assets/120410992/34dae3f5-1e15-4864-b7b9-c578d0751300">
<img width="1089" alt="Screenshot 2023-06-08 at 10 16 15" src="https://github.com/mysociety/whatdotheyknow-theme/assets/120410992/2564a25b-5984-4640-88d7-5a443ce86945">

## Notes to reviewer
I added some basic styling to the table, but that's entirely optional.

mdeuk commented on lib/views/help/performance_tags.html.erb:
```diff
@@ -0,0 +1,131 @@
+            <td><a href="/search/tag:fee_scare/requests"><%= pluralize(InfoRequest.is_public.with_tag('fee_scare').size, 'Request') %> tagged <code>fee_scare</code></a></td>
+        </tr>
+        <tr>
+            <td>Authority refuses to recognise valid FOI requests</td>
```
```suggestion
            <td>Authority refuses to recognise valid <abbr>FOI</abbr> requests</td>
```

mdeuk reviewed (approved):
Hey, this looks great! 👌

It's good to see this metadata being more prominently displayed.

I've left a set of suggested changes - nothing too contentious, mainly adding `bold` tags to the metrics[^1], and `abbr` tags for abbreviations.

I've also added a really basic CTA with a link out to the Research folks at the bottom. Please feel free to revise as necessary, copywriting isn't my best skill.

[^1]: I did debate whether the links, themselves, ought to start after the metric, but I think it's fine to use the way it is… adding the bold element will make it stand out.

HelenWDTK commented:
Thanks @mdeuk - all great suggestions as usual 🙂

HelenWDTK commented:
I've changed the url as suggested (performance tracking seemed best). I've added pit_delay to the table, and removed the tags that were not to do with authority performance, but more to do with site management (we can add them back later if we'd like). I've also added some text explaining when we tag stuff, and a disclaimer at the end.

garethrees reviewed (approved):
🚀
